public/js/discover/searchList.js

```diff
@@ -3,17 +3,17 @@ const searchCardTemplate = `
         <div class="search-card" id="{{this.id}}">
           <div class="search-left">
             <div class="search-dropoff-info">
-              <h1 class="search-title">{{this.title}}</h1>
+              <h1 class="search-title">{{this.name}}</h1>
               <h2 class="search-address">{{this.address}}</h2>
               <h2 class="search-phone">{{this.phone}}</h2>
             </div>
             <div class="search-dropoff-options">
               {{#*inline "acceptanceSymbol"}}
                 {{#if accepts}} <span class="valid-symbol">✓</span> {{else}} <span class="invalid-symbol">✗</span> {{/if}}
               {{/inline}}
-              <p>Drop off {{> acceptanceSymbol accepts=this.supportsDropOff}} </p>
-              <p>Mail in {{> acceptanceSymbol accepts=this.supportsMailIn}} </p>
-              <p>Pick up {{> acceptanceSymbol accepts=this.supportsPickUp}} </p>
+              <p>Drop off {{> acceptanceSymbol accepts=this.acceptsDropOff}} </p>
+              <p>Mail in {{> acceptanceSymbol accepts=this.acceptsMailIn}} </p>
```

Comment:
This would be `this.acceptsShipping` changed from the "mail in" wording that had a conotation of monetary donation mailed in instead of shipping an item.